README.md

```diff
@@ -16,13 +16,26 @@ Replace the connection string URL with your ElasticSearch URL and add the elmah
 
 That's it, dudes!
 
-If you don't like the default generated index named _elmah_, you should override it by specifying a default index name:
-
-    <errorLog
-        type="Elmah.Io.ElasticSearch.ElasticSearchErrorLog, Elmah.Io.ElasticSearch"
-        connectionStringName="ElmahIoElasticSearch"
-        defaultIndex="myindex" />
+The Elastic index name can be specified one of two ways, let's say you want the index name to be elmahCurrent:
```

Comment:
updated documentation, we implemented this easier way to specify the ES index name a while ago but never updated the documentation.